src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -413,6 +425,8 @@ void DlgPrefDeck::slotResetToDefaults() {
     // Don't load tracks into playing decks.
     checkBoxDisallowLoadToPlayingDeck->setChecked(true);
 
+    // Clone decks by double-tapping Load button.
+    checkBoxCloneDeckOnLoadDoubleTap->setChecked(true);
```

Comment:
it would be nice to use a constant for the default true

Reply:
please explain what you mean.

Reply:
In a anonymous namespace:

contexpr bool kDefaultCloneDeckOnLoad = true;

This helps to set the default value only in one place.